src/qtism/runtime/storage/binary/QtiBinaryStreamAccess.php

```diff
@@ -687,7 +689,9 @@ public function writeIntOrIdentifier($intOrIdentifier)
                 $this->writeBoolean(false);
                 $this->writeString($intOrIdentifier);
             } else {
-                $msg = "The intOrIdentifier value to be written must be an integer or a string, '" . gettype($intOrIdentifier) . "' given.";
+                $msg = "The intOrIdentifier value to be written must be an integer or a string, '"
+                       . gettype($intOrIdentifier)
+                       . "' given.";
```

Comment:
i guess `sprintf` can help make it easy understandable
```suggestion
                $msg = sprintf(
                       "The intOrIdentifier value to be written must be an integer or a string, '%s given.",
                       gettype($intOrIdentifier)
                );
```